src/transform_all.bat

```diff
@@ -24,6 +24,8 @@ set file_name=!file_name:~0,-3!.%extension%
 echo:  \--^> !file_name!    
 echo "%COMMONPROGRAMFILES(x86)%\microsoft shared\TextTemplating\11.0\TextTransform.exe" -out !file_name! %%d
 "%COMMONPROGRAMFILES(x86)%\microsoft shared\TextTemplating\11.0\TextTransform.exe" -out !file_name! %%d
+echo "%COMMONPROGRAMFILES(x86)%\microsoft shared\TextTemplating\12.0\TextTransform.exe" -out !file_name! %%d
```

Comment:
Can we specify version 12.0 as a parameter, and pass the correct parameter based on Visual studio version from the cs project.